developerportal/apps/topics/models.py

```diff
@@ -130,7 +141,16 @@ class Topic(BasePage):
     )
 
     # Meta
-    icon = FileField(upload_to="topics/icons", blank=True, default="")
+    icon = FileField(
+        upload_to="topics/icons",
+        blank=True,
+        default="",
+        help_text=(
+            "MUST be a black-on-transparent SVG icon ONLY, "
+            "with no bitmap embedded in it."
+        ),
+        validators=[check_for_svg_file],
```

Comment:
Not sure if it helps, but Django ships with a [FileExtensionValidator](https://docs.djangoproject.com/en/2.2/ref/validators/#fileextensionvalidator)

Reply:
Ah, a 2.1 addition I hadn't noticed - cheers!